### ISSUE TITLE
Ability to store deployment properties within SCDF database

### ISSUE THREAD

**Issue by mbhuie** [spring-cloud/spring-cloud-dataflow#1875]
Can the deployment properties be stored within SCDF instead of having to specify it each time we deploy a stream? This is really annoying for operations team as they have to manually copy and paste it each time from say a confluence page or file. It can also be error prone e.g. if they import a wrong file or copy-paste wrong deployment property

Perhaps the deployment properties could be asked first time and everything in the text area can be restored from an SCDF table next time onward. An operator can change it if he wants to or simply accept it as it is.

**Comment by ilayaperumalg** [comment]
Hi @mbhuie , 

Thanks for reaching out!
This has been identified and being tracked in https://github.com/spring-cloud/spring-cloud-dataflow/issues/1738

**Comment by mbhuie** [comment]
Looks like Skipper is only available for CF, Kubernates. We are using YARN for our solution.

**Comment by sabbyanandan** [comment]
Hi, @mbhuie. As of spring-cloud/spring-cloud-dataflow#1882, we now persist deployment properties both for classic and skipper modes - see example below.

```
dataflow:>stream deploy --name foo --properties "deployer.log.count=2"
Deployment request has been sent for stream 'foo'

dataflow:>stream info --name foo
╔══════════════════════════════╤══════════════════════════════╤══════════════════════════════╗
║             Name             │             DSL              │            Status            ║
╠══════════════════════════════╪══════════════════════════════╪══════════════════════════════╣
║foo                           │time | log                    │deployed                      ║
╚══════════════════════════════╧══════════════════════════════╧══════════════════════════════╝

Stream Deployment properties: {
  "log" : {
    "spring.cloud.deployer.indexed" : "true",
    "spring.cloud.deployer.count" : "2",
    "spring.cloud.deployer.group" : "foo"
  },
  "time" : {
    "spring.cloud.deployer.group" : "foo"
  }
}
```

This is available in 1.3 BUILD-SNAPSHOTs; please give it a go and let us know what you think. 

Thanks for the continued interest!

**Comment by mbhuie** [comment]
Thanks. 

We are using Spring Cloud Dataflow for YARN, seems like that project is not maintained any more.
